graph.js

```diff
@@ -59,6 +60,24 @@ function outputDeployConfig(cmd) {
   console.error('')
 }
 
+function outputAuthConfig(node, authToken) {
```

Comment:
We don't need this. If `graph auth` only has two inputs and if anything is wrong we should handle/report that in a more compact way.

Reply:
I take this back, but I'd like to request changes here:

* Change `Node:` to `Graph node:` to be consistent with the other outputs.
* Rename `authToken` to `accessToken`
* Change `AuthToken` to `Access token`
* Change `No auth token defined` to `Missing`
* Change `The auth token provided is too long` to `Access token is too long`
* Remove `AuthToken: Hidden` entirely